src/main/java/mocacong/server/dto/request/CafeRegisterRequest.java

```diff
@@ -0,0 +1,19 @@
+package mocacong.server.dto.request;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Getter
+public class CafeRegisterRequest {
+    @NotBlank
```

Comment:
예외코드랑 예외메시지 지정해주면 더 명확해질 듯해요
```suggestion
    @NotBlank(message = "1012:공백일 수 없습니다.")
```